cli/src/auth/oidc.rs

```diff
@@ -132,6 +133,10 @@ pub fn build_auth_url(
         .append_pair("scope", &OIDC_SCOPES.join(" "))
         .append_pair("state", state.as_ref());
 
+    if matches!(*action, AuthAction::Reauth) {
```

Comment:
Would be cleaner if we just implemented PartialEq+Eq for `AuthAction` which is trivial to do in our case.